src/components/common/ThemeSelector.js

```diff
@@ -0,0 +1,82 @@
+import Tippy from '@tippyjs/react';
+import classNames from 'classnames';
+import { useState } from 'react';
+import { LuMoon, LuSun } from 'react-icons/lu';
+import { MdComputer } from 'react-icons/md';
+
+export default function ThemeSelector() {
+  const [activeTheme, setActiveTheme] = useState(localStorage.theme || '');
+
+  function onThemeChange(theme) {
+    setActiveTheme(theme);
+
+    if (theme === '') {
+      localStorage.removeItem('theme');
+    } else {
+      localStorage.theme = theme;
+    }
+
+    if (
+      localStorage.theme === 'dark' ||
+      (!('theme' in localStorage) &&
+        window.matchMedia('(prefers-color-scheme: dark)').matches)
+    ) {
+      document.documentElement.classList.add('dark');
+    } else {
+      document.documentElement.classList.remove('dark');
+    }
+  }
+
+  const themeButtonStyle =
```

Comment:
The buttons feel maybe a touch tall in comparsion to the other buttons in the interface (but esp the graph selectors).